### PR TITLE
Revert "Bug/to approve endpoint"

### PR DESCRIPTION
**Pull request by RyderBlues**
Reverts Lambda-School-Labs/human-rights-first-police-fe-a#326

### CODE REVIEW

**Review by devin-mitchell — APPROVED** [review]
oopsie

**Review by nickcabuno — APPROVED** [review]
^

File: src/components/form/TwitterForm.js
Comment (ahmedseragcodes) [inline src/components/form/TwitterForm.js:25]: Hmm is this get request empty then?

File: src/components/AdminDashboard/AdminDashboard.js
Comment (ahmedseragcodes) [inline src/components/AdminDashboard/AdminDashboard.js]: Got confirmation that the deployed DS endpoints via swagger are HTTP not HTTPS, so this is fine

**Comment by ahmedseragcodes** [comment]
Resolved merge conflict, leaving it up to release manager to merge or not bc old PR, if was key, likely should have been merged before.

**Review by ahmedseragcodes — APPROVED** [review]
Confirmed URL has been correctly changed, resolved merge conflict. Approved after testing URL responds properly. Leaving up to release manager to merge or not because it is an older PR.

File: src/components/AdminDashboard/AdminDashboard.js
Comment (ChristopherKchilton) [inline src/components/AdminDashboard/AdminDashboard.js:107]: After trying both addresses it seems the AWS is currently only set up for http use. At this time, this would probably be the bug fix to this problem.  However it may be advisable to look into AWS to see how we could set up https.

Comment (paulstgermain) [reply src/components/AdminDashboard/AdminDashboard.js:107]: Seconding this opinion - AWS is seemingly currently configured for HTTP use, but in the future, it may be worthwhile to look into configuring for HTTPS for added security.

**Review by paulstgermain — APPROVED** [review]
This is a relatively small change that seeks to align the get request with the AWS database and most recent DS API endpoint. After testing to confirm this endpoint is functional and considering the size of the PR, approving this PR.

A couple of notes for future PRs though! In the future, ensure that the PR description is more sufficiently filled out - leave detail regarding the problem this PR is attempting to resolve, as well as an explanation of the route you ook to solve that problem via this PR.

Additionally, this PR was before the roll-out of the new Labs requirements - but use this as an example: Make sure not to forget to record a Loom video of yourselves explaining the PR & code in further detail! An easy way to do this is to install the Loom Chrome extension -this will enable an option on GitHub itself to record a Loom video when opening a PR.

**Pull request merged**
